README.md

```diff
@@ -353,6 +354,7 @@ A curated list of awesome Java frameworks, libraries and software.
 
 * [JDK 9](https://jdk9.java.net/) - Early access releases of JDK 9.
 * [OpenJDK](http://openjdk.java.net/) - Open-source implementation.
+* [IBMJDK](https://www.ibm.com/developerworks/java/jdk/) -- IBM version of JDK with a high performance JVM and customized libraries.
```

Comment:
We deleted it since the usage isn't really recommendable. Also there are better (commercial) options for high-performance JDKs.